test/src/navigation_state_test.dart

```diff
@@ -1,29 +1,33 @@
 import 'package:flutter_redux_navigation/flutter_redux_navigation.dart';
+import 'package:flutter_redux_navigation/src/navigation_destination.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 void main() {
-  test('should have null paths as initial state', () {
```

Comment:
Please keep those tests and create additional ones for `NavigationDestination`. We need to make sure we do not break current functionality.

Maybe create separate `group`s.

Reply:
I experimented with separate groups, but it causes a lot of code duplications in the tests, so I decided against it for now.